README.md

```diff
@@ -121,12 +143,18 @@ even if it involves multiple applications or ETL pipelines.
 
 ## How to generate Code coverage report
 ```sbt
-sbt jacoco
+sbt "project agent" ++2.13.11 jacoco
+```
+```sbt
+sbt "project model" ++2.13.11 jacoco
```

Comment:
++ does not work when used with projectMatrix

Reply:
I have not problem to remove it. @benedeki  ???

Reply:
Yes, working on it's removal. Don't mention in `README.md`

Reply:
Removed in commit - c4e02062b25ec867fa048f360dd333c706e7cc03.